locales/locale.hu.ts

```diff
@@ -5,97 +5,97 @@ export const locale: Locale = {
   language: 'hu',
   items: [
     // General
-    { source: 'Save', target: 'Megment' },
-    { source: 'Create', target: 'Teremt' },
-    { source: 'Duplicating', target: 'Másolás' },
-    { source: 'Duplicate', target: 'Másolat' },
-    { source: 'Mass Edit', target: 'Mass Edit' },
+    { source: 'Save', target: 'Ment' },
+    { source: 'Create', target: 'Létrehoz' },
+    { source: 'Duplicating', target: 'Másol' },
+    { source: 'Duplicate', target: 'Duplikál' },
+    { source: 'Mass Edit', target: 'Csoportos szerkesztés' },
     { source: 'Export', target: 'Export' },
-    { source: 'Import', target: 'Importálás' },
+    { source: 'Import', target: 'Import' },
     { source: 'Delete', target: 'Töröl' },
 
     // Fields
     { source: 'optional', target: 'választható' },
-    { source: 'choose', target: 'választ' },
-    { source: 'Clear value', target: 'Tiszta érték' },
+    { source: 'choose', target: 'válasszon' },
+    { source: 'Clear value', target: 'Érték törlése' }, 
     { source: 'loading...', target: 'Betöltés...' },
     { source: 'Nothing found', target: 'Nincs találat' },
     { source: 'Search...', target: 'Keresés...' },
-    { source: 'Enter at least {0} characters', target: 'Írjon be legalább {0} karaktert' },
+    { source: 'Enter at least {0} characters', target: 'Írjon be legalább {0} karaktert!' },
```

Comment:
Why adding exclamation mark? There is no such in original

Reply:
The reasoning was that grammatically correct imperative sentences in Hungarian should end with an exclamation mark.